java/zulu-jdk8/Portfile

```diff
@@ -0,0 +1,205 @@
+
+    set jdk_version(x64)       14.0.2
+    set zulu_version(x64)      14.29.23
+    set major                  14
+    revision                   0
+
+    supported_archs            x86_64
+
+    checksums                  zulu${zulu_version(x64)}-ca-jdk${jdk_version(x64)}-macosx_x64.zip \
+        rmd160                 f97f9d7d7ad526797ff8dcd32ab9c5187aa4f5cb \
+        sha256                 57e737d639a6927c82626fd18257751e6bb4d64e291b8fac62ddad01255a1d77 \
+        size                   207255111
+}
+
+subport zulu-jdk15 {
+
+    set jdk_version(x64)       15.0.1
+    set zulu_version(x64)      15.28.51
+    set major                  15
+    revision                   0
+
+    supported_archs            x86_64
+
+    checksums                  zulu${zulu_version(x64)}-ca-jdk${jdk_version(x64)}-macosx_x64.zip \
+        rmd160                 20f1b78a26beab1cd71c2fd99107704df53fc3e7 \
+        sha256                 084c67b6ada87e8ba1f8abf35af2c8a56b204ccf513600af6e09ae7880824a84 \
+        size                   206369908
+}
+
+categories       java devel
+maintainers      nomaintainer
+platforms        darwin
+license          GPL-2
+homepage         https://www.azul.com/downloads/zulu-community/
+
+
+set zulu_arch(arm64)       aarch64
+set zulu_arch(x86_64)      x64
+
+if {![info exists zulu_arch(${build_arch})]} {
+    set zulu_arch(${build_arch}) ${build_arch}
+}
+
+set zulu_name(arm64)       macos
+set zulu_name(x86_64)      macosx
+
+if {![info exists zulu_arch(${build_arch})]} {
+    set zulu_arch(${build_arch}) macosx
+}
+
+if {${os.platform} eq "darwin" && ${os.major} < 14} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} ${version} is only supported on Mac OS X 10.10 Yosemite or later."
+        return -code error
+    }
+}
+
+description  Open Java Development Kit ${major} (Zulu) with HotSpot VM
+long_description OpenJDK build provided by Azul, built from a fully \
+    open-source set of build scripts and infrastructure. \
+    \
+    HotSpot is the VM from the OpenJDK community and  the most widely used VM. \
+    It is suitable for all workloads.
+
+set version  $zulu_version($zulu_arch($build_arch))
+
+master_sites https://cdn.azul.com/zulu/bin/
+distname     zulu$zulu_version($zulu_arch($build_arch))-ca-jdk$jdk_version($zulu_arch($build_arch))-$zulu_name($build_arch)_$zulu_arch($build_arch)
+use_zip      yes
+worksrcdir   zulu$zulu_version($zulu_arch($build_arch))
+configure.cxx_stdlib libstdc++
+
+use_configure    no
+
+build {}
+
+# macOS Java tools expect to find Java virtual machines under /Library/Java/JavaVirtualMachines, which is not under ${prefix}.
+destroot.violate_mtree yes
+
+set target /Library/Java/JavaVirtualMachines/${subport}
+set destroot_target ${destroot}${target}
+
+destroot {
+    xinstall -m 755 -d ${destroot_target}
+    copy ${worksrcpath}/zulu-${major}.jdk/Contents ${destroot_target}
+}
+
+notes "
+If you have more than one JDK installed you can make ${subport} the default
```

Comment:
```suggestion
If you have more than one JDK installed you can make ${subport} the default\
```

Reply:
why?! it is multi line.

Reply:
Add the backslash as I indicated, so that MacPorts can handle wrapping the text to the user's terminal width.